request_a_govuk_domain/request/admin.py

```diff
@@ -148,23 +152,57 @@ class ReviewInline(admin.StackedInline):
     verbose_name_plural = "Reviews"
 
 
+def build_registration_data_from_application(application):
```

Comment:
Could you add type hints (also in other functions you added or modified)?

Reply:
Done